packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -941,6 +943,16 @@ mixin WidgetInspectorService {
     _errorsSinceReload = 0;
   }
 
+  bool setStructuredErrorsEarly() {
```

Comment:
this should be named
isStructuredErrorsEnabled or similar not set. The word set in the name implies a value is set.